README.md

```diff
@@ -57,6 +57,7 @@ If the image has a dark alternative, that's what you will get otherwise it will
 | links.reddit        | `String` | `false`    |         |                                                           |
 | links.youtube       | `String` | `false`    |         |                                                           |
 | links.instagram     | `String` | `false`    |         |                                                           |
+| donations           | `Object` | `false`    |         |                                                           |
```

Comment:
The description should be a little more specific about how it should look like:
`{ "ethereum": "0x...", "zilliqa": "zil1..." }`